app/product/queries/searchProducts.ts

```diff
@@ -50,6 +51,26 @@ const searchProducts = resolver.pipe(
       orderBy: order,
     }
 
+    const queriesWithTag: Prisma.ProductFindManyArgs = {
+      take: input.take,
+      skip: input.skip,
+      where: {
+        name: {
+          contains: input.name,
```

Comment:
```suggestion
          contains: input.name,
          mode: 'insensitive',
```
Same for the above query